inlang/source-code/sdk/src/createMessagesQuery.ts

```diff
@@ -365,6 +371,13 @@ async function loadMessagesViaPlugin(
 				// we don't have to check - done before hand if (messages.has(loadedMessageClone.id)) return false
 				messages.set(loadedMessageClone.id, loadedMessageClone)
 				messageState.messageLoadHash[loadedMessageClone.id] = importedEnecoded
+				loadedMessageCount++
+			}
+			if (loadedMessageCount % messagesPerTick === 0) {
+				// move loading of the next messages to the next ticks to allow solid to cleanup resources
+				// solid needs some time to settle and clean up
+				// https://github.com/solidjs-community/solid-primitives/blob/9ca76a47ffa2172770e075a90695cf933da0ff48/packages/trigger/src/index.ts#L64
+				await 0
```

Comment:
I think I'd prefer to use a setTimeout here.

Reply:
I verified that the behavior of sleep(0) with `setTimeout(...,0)` is not the same as `await 0`